dts/include/coronet/viper-poe-80W.dtsi

```diff
@@ -0,0 +1,7 @@
+#include "viper-poe.dtsi"
+
+&soc {
+	weos-poe {
```

Comment:
This is not WeOS, so we should probably not reference that here.  And even in WeOS it should probably have a more generic name since this is also the first generation PoE, right?  Possibly coronet-poe or some generic name shared with other platforms.